docs/usage/post_training_compression/weights_compression/Usage.md

```diff
@@ -61,6 +61,15 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset) # model is openvino.Model object
 ```
 
+- Additionally, it is possible to use the model to generate a synthetic dataset based on the model output. For this purpose the `nncf.data.generate_text_data` method exits. This method returns the list of the strings generated by the model. Generation takes time and depends on various conditions like model size, requested dataset length or environment setup. Also, since the dataset was generated by model output, it does not guarantee significant accuracy improvement. This method is recommended to use only in case when any better dataset option is not available. Refer to the [example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_synthetic_data/) for the details of the usage.
```

Comment:
I suggest to slightly rephrase it to:
Additionally, it is possible to generate a synthetic dataset by `nncf.data.generate_text_data` method to use it in the data-aware weight compression. The method takes a language model (e.g. from optimum.intel.openvino) and a tokenizer (e.g. from transformers) as input and returns the list of strings generated by the model. Note that the dataset generation takes time and depends on various conditions like the model size, requested dataset length or environment setup. Also, since the dataset is generated by the model output, it does not guarantee significant accuracy improvement after the compression. This method is recommended only in cases when a better dataset is not available. Refer to the [example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_synthetic_data/) for details of the usage.

Reply:
Done.